lib/github.js

```diff
@@ -128,7 +128,12 @@ function replaceURL(url, options) {
     });
 }
 
+function setToken(t) {
```

Comment:
Rather than exposing this, you might try letting `exports = function(token) { return {get, post, put, patch} }` so that the way to use the module is `require('github')(token)`. There are many things you could vary about that suggestion like making it `new`able, options as dictionary, and so on, but the basic point is to make the code that needs a token unreachable until you provide it.

Reply:
Sadly `github.js` is `require`d from a bunch of library files and other scripts currently, none of which would have the `token` available unless we added `.json` parsing to all of them. Long term I would plan to clean this up, but for now I'm trying to make the smallest change possible to get this running on appengine. SGTY? 

I did change `var token;` back to `var token = process.env.GITHUB_TOKEN;`, which sort of lets other scripts use `github.js` as long as they have env support.